isIsogram.js

```diff
@@ -0,0 +1,13 @@
+export function isIsogram(str) {
+    for (let i = 0; i < str.length; i++) {
```

Comment:
Did you consider calling your loop indexes differently? Right now it's `i` and `j`. 
When you read this code you need to THINK what each mean.

Imagine you called one `stringCharacterIndex` and `substringCharacterIndex` (not saying it's a best names, just an example). Then your code will make more sense:
```
if (strNoCase.charAt(stringCharacterIndex) === strNoCase.charAt(substringCharacterIndex)) {
  \\do stuff
}
```
What do you think?